tip/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:3.9-slim-bullseye
```

Comment:
Is Debian (slim) required or can it be Alpine too?

Any restrictions to not use Python 3.10?

Reply:
In fact, Alpine was used until 17 days ago. Then, we changed to Debian (slim) and we found out that it is faster indeed. PyGitHub needs to be installed, and the installation on Debian is faster (because it can use a wheel). See https://github.com/eine/tip/issues/160#issuecomment-969128994.

Changing from Alpine to Debian did not fix the connection issues, so 12 days ago we tried downgrading to Python 3.9: https://github.com/eine/tip/issues/160#issuecomment-974620079. We should maybe stay with 3.9 until we find out what's going on.

Reply:
What do you mean by *faster*:
* execution speed, or
* setup/installation speed?

Ok, then Debian is OK, I was thinking Alpine is smaller and doesn't need the bigger Debian slim image.

Python 3.9 is OK for now.
Is `lxml` used? I habe problems until a few days ago with `lxml` in Python 3.10.

Reply:
With faster I mean installation speed. However, currently that is also execution speed because the Conatiner Action is built in each run (that's how GitHub's Container Actions work).

I don't think lxml is used, but I'm unsure.